python/pyspark/context.py

```diff
@@ -592,22 +664,28 @@ def range(
         way as python's built-in range() function. If called with a single argument,
         the argument is interpreted as `end`, and `start` is set to 0.
 
+        .. versionadded:: 1.5.0
+
         Parameters
         ----------
         start : int
             the start value
-        end : int, optional
+        end : int, optional, default None
             the end value (exclusive)
-        step : int, optional
-            the incremental step (default: 1)
-        numSlices : int, optional
+        step : int, optional, default 1
+            the incremental step
+        numSlices : int, optional, default None
             the number of partitions of the new RDD
 
         Returns
         -------
         :py:class:`pyspark.RDD`
             An RDD of int
 
+        See Also
+        --------
+        :meth:`SparkSession.range`
```

Comment:
```suggestion
        :meth:`pyspark.sql.SparkSession.range`
```

Haven't built the docs against this PR but you would probably need the fully qualified path because this class is not being imported within this file.